examples/run_field_dep_justMw.py

```diff
@@ -191,7 +186,7 @@
         .set_units("indirect", "scan #")
     )
 sweep_data.name(config_dict["type"] + "_" + str(config_dict["field_counter"]))
-sweep_data.set_prop("postproc_type", "field_sweep_v2")
+sweep_data.set_prop("postproc_type", "field_sweep_v3")
```

Comment:
increment post processing version so that we aren't multiplying by nScans anymore